index.html

```diff
@@ -110,21 +78,18 @@ <h2><b>Tonic</b></h2>
           <li><a  class="button" href="#">CSS</a> </li>
           <li><a  class="button" href="#">javaScript</a></li>      
         </ul>
-    
-
+  
         <div class="action"> 
           <form class="normalbutton">
             <a class="seeproject"  href="#">See Project</a> 
           </form> 
```

Comment:
- [ ] [OPTIONAL] I think it will be semantically more appropriate if you use a button tag rather than a for tag for the buttons in the page, scrrenreaders will thank you :pray: 
- [ ] [OPTIONAL] Also I think the button interactions is not what it is in the figma design currently it looks like this :point_down: 
![image](https://user-images.githubusercontent.com/97521557/208252656-dd528c5b-e7c7-4d3f-8dbc-ddbdc9506447.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/208252603-e483fa78-1407-4443-b05a-aaf48af86817.png)